src/plots/plot.ts

```diff
@@ -67,7 +67,22 @@ export class Plot extends Component {
     this.animator(Plots.Animator.RESET, new Animators.Null());
   }
 
-  public anchor(selection: SimpleSelection<void>) {
+  private _canvas: d3.Selection<HTMLCanvasElement, any, any, any>;
+  private _createCanvas() {
+    const element = this.element();
+    this._canvas = element.append<HTMLCanvasElement>("canvas");
```

Comment:
Then those things can create canvases. This is no different than how existing Components manage their own drawing areas/d3 selections